tests/cross_functional/ui/fusion_access/test_virtualmachine_lifecycle.py

```
@@ -35,39 +43,237 @@ class TestVirtualMachineLifecycle(ManageTest):
+            if line and not line.startswith("md5sum") and len(line.split()) >= 2:
+                candidate = line.split()[0]
+                if len(candidate) == 32 and all(
+                    c in "0123456789abcdefABCDEF" for c in candidate
+                ):
+                    md5sum_output = candidate
+                    break
+
+        logger.info(f"Parsed md5sum checksum: {md5sum_output}")
+        return md5sum_output
+
+    def _wait_for_vmi_agent_connected(self, vm_name, namespace, timeout=600):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_wait_for_vmi_agent_connected`

Add return type annotation: `None`

(ANN202)